scala-package/infer/src/test/java/org/apache/mxnet/infer/javaapi/ObjectDetectorOutputTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnet.infer.javaapi;
+
+import org.junit.Test;
+
+public class ObjectDetectorOutputTest {
+
+    @Test
+    public void testConstructor() {
+
+        float[] arr = new float[]{0f, 1f, 2f, 3f, 4f};
+
+        ObjectDetectorOutput odOutput = new ObjectDetectorOutput("simba", arr);
+
+        assert (odOutput.getClassName().equals("simba"));
+        assert (odOutput.getProbability() == 0);
```

Comment:
Two comments
1) You shouldn't compare floats with equality. The recommended way is to use thresholds. https://howtodoinjava.com/java/basics/correctly-compare-float-double/
2) Use assertEquals with a delta to do this for you. Additionally, if there is inequality, assertequals will provide a more useful error message.

Reply:
Good point. I am now using assertEquals with delta. 
41f33716118c7cad36605daad820cc97a2e46201